eve/main.yml

```diff
@@ -277,7 +284,34 @@ stages:
       - Upload:
           name: upload artifacts
           source: sosreport
```

Comment:
Like in Upgrade please move `upload artifacts` and sosreport to the end of the step (`alwaysRun:true` so we will always have the report

Reply:
I wanted to have 2 sos report collections if someday we want to see the difference between the two states

Reply:
Then please rename the file like `sosrepot-blabla-post-install.tar.xz` `sosreport-blabla-post-upgrade.tar.xz` `sosreport-blabla-post-downgrade`